projects/media-viewer/src/lib/media-viewer/viewers/image-viewer/image-viewer.component.spec.ts

```diff
@@ -13,7 +13,7 @@ describe('ImageViewerComponent', () => {
     const mockRenderer = new MockRenderer();
 
   beforeEach(async(() => {
-    TestBed.configureTestingModule({
+    return TestBed.configureTestingModule({
```

Comment:
is the return needed here?

Reply:
intellij threw a warning and I assumed the wrapped async function would want to wait for that promise.